_pages/who.md

```diff
@@ -4,7 +4,10 @@ permalink: /who/
 title: "Who we are"
 breadcrumb: "about"
 grid-section:
-  - group-heading: "Steering Committee Members"
+  - group-heading: "Measurement Lab Staff"
```

Comment:
without the content here, we probably should've have the header yet.

Reply:
missed that heading. removed in latest commit.